### PR TITLE
Release 0.0.28

### PR DESCRIPTION
**Pull request by vontell**

### CODE REVIEW

**Review by batu — APPROVED** [review]

**Review by addisonbgross — APPROVED** [review]

File: docs/changelog.mdx
Comment (RG-nAmKcAz) [inline docs/changelog.mdx:23]: `now requires authentication` -> is fine, but we should mention that this explicitly requires using API keys / having an API key setup in the SDK

File: docs/changelog.mdx
Comment (RG-nAmKcAz) [inline docs/changelog.mdx:27]:
```suggestion
- Data for the last segment of a recording is now properly saved
```

File: docs/changelog.mdx
Comment (RG-nAmKcAz) [inline docs/changelog.mdx:26]:
```suggestion
- Fixed issues causing the game to freeze for a few seconds when starting
```

**Review by RG-nAmKcAz — CHANGES_REQUESTED** [review]

File: docs/changelog.mdx
Comment (vontell) [reply docs/changelog.mdx]: Added

**Review by RG-nAmKcAz — APPROVED** [review]

**Comment by vontell** [comment]
@abeizer will wait for your review before merging

**Review by abeizer — APPROVED** [review]

**Pull request merged**
